source/Octopus.Client/Editors/Async/MachineEditor.cs

```diff
@@ -54,20 +55,29 @@ public MachineEditor(IMachineRepository repository)
             EnvironmentResource[] environments,
             string[] roles,
             TenantResource[] tenants,
-            TagResource[] tenantTags)
+            TagResource[] tenantTags, 
+            TenantedDeploymentParticipation? tenantedDeploymentParticipation = null)
```

Comment:
My thinking was that making it nullable meant existing code won't break.  Otherwise it will cause a compile-error when upgrading.